byway/src/main/java/com/google/sps/data/Trip.java

```diff
@@ -134,4 +144,18 @@ public static Trip fromEntity(Entity tripEntity) {
             "Trip entity does not contain route");
     return new Trip(keyString, start, destinations, interests, route);
   }
+
+  /**
+   * Creates a Trip entity based on the Trip class attributes
+   *
+   * @return tripEntity entity from datastore
```

Comment:
this `@return` is kind of inaccurate, since the returned entity is newly-constructed, not from the datastore at all.

`@return an {@link Entity} with properties set from this {@link Trip} instance, which can be put in the datastore.`

Reply:
Oh right, bad description there. Changed.